lib/board.rb

```diff
@@ -0,0 +1,12 @@
+class Board
+  attr_accessor :board_grid
+
+  ##initialize board
```

Comment:
I personally would prefer not to see any comments committed, **unless** they are vital to future developers gaining clarification on a very specific circumstance, which cannot be explained with good naming.

For example - here your method name is `initialize` and it's within a `board` class, so the comment doesn't add any extra information!